.travis.yml

```diff
@@ -94,8 +101,13 @@ script:
 - if [ $RUN_E2E_TESTS_EXTENSIONS == 'true' ]; then travis_retry bash scripts/run_e2e_tests.sh --suite="extensions" --prod_env; fi
 - if [ $RUN_E2E_TESTS_LEARNER_DASHBOARD == 'true' ]; then travis_retry bash scripts/run_e2e_tests.sh --suite="learnerDashboard" --prod_env; fi
 - if [ $RUN_E2E_TESTS_LIBRARY == 'true' ]; then travis_retry bash scripts/run_e2e_tests.sh --suite="library" --prod_env; fi
+- if [ $RUN_E2E_TESTS_RATINGS == 'true' ]; then travis_retry bash scripts/run_e2e_tests.sh --suite="ratings" --prod_env; fi
+- if [ $RUN_E2E_TESTS_SUBSCRIPTIONS == 'true' ]; then travis_retry bash scripts/run_e2e_tests.sh --suite="subscriptions" --prod_env; fi
 - if [ $RUN_E2E_TESTS_USERS == 'true' ]; then travis_retry bash scripts/run_e2e_tests.sh --suite="users" --prod_env; fi
-
+# Run e2e tests for mobile in dev mode since the demo explorations and collections are only available in the dev mode and not in production mode.
+#- if [ $RUN_E2E_TESTS_MOBILE == 'true' ] && [ $TRAVIS_BRANCH == 'develop' ] && [ $TRAVIS_EVENT_TYPE == 'push' ] && [ $TRAVIS_PULL_REQUEST == 'false' ]; then travis_retry bash scripts/run_e2e_tests.sh --browserstack; fi
```

Comment:
Go ahead and revert back to this.

Reply:
Will merge also count as a push for event type?

Reply:
Yes, I think so since there are only four event types: push, pull_request, api and cron.